internal/services/instance/servers_data_source.go

```diff
@@ -203,10 +205,10 @@ func DataSourceInstanceServersRead(ctx context.Context, d *schema.ResourceData,
 			rawServer["placement_group_id"] = zonal.NewID(zone, server.PlacementGroup.ID).String()
 			rawServer["placement_group_policy_respected"] = server.PlacementGroup.PolicyRespected
 		}
-		if server.IPv6 != nil {
-			rawServer["ipv6_address"] = server.IPv6.Address.String()
-			rawServer["ipv6_gateway"] = server.IPv6.Gateway.String()
-			prefixLength, err := strconv.Atoi(server.IPv6.Netmask)
+		if server.IPv6 != nil { //nolint:staticcheck
```

Comment:
We need to add test and documentation to ensure that we detect when this field is removed from API responses and that the user are warned that the field is deprecated.